product_contract_variable_quantity/README.rst

```diff
@@ -0,0 +1,87 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Product Contract Variable Quantity
+==================================
+
+Product contract with variable quantity
+
+Installation
+============
+
+To install this module, you need to:
```

Comment:
Complete README

Reply:
There readme will be overridden after merge.

You mean add more info to readme fragments?